pkg/provider/provider.go

```diff
@@ -51,44 +51,65 @@ const (
 )
 
 type PodWrapper struct {
```

Comment:
As our custom container wrapper is called simply "Container", cannot we just simply name this one "Pod" ?

Reply:
fixed